packages/aragon-cli/test/lib/apm/grantNewVersionsPermission.test.js

```diff
@@ -0,0 +1,285 @@
+import test from 'ava'
+import sinon from 'sinon'
+import proxyquire from 'proxyquire'
+
+/* Default values */
+
+const apmRepoName = 'test.aragonpm.eth'
+const apmOptions = {}
+apmOptions['ens-registry'] = '0x1234512345123451234512345123451234512345'
+const gasPrice = 1
+const txOptions = { gasPrice }
+const grantees = ['0x1234512345123451234512345123451234512345']
+const progressHandler = () => {}
+
+/* Setup and cleanup */
+
+test.beforeEach('setup', t => {
+  const web3Stub = {
+    eth: {
+      getAccounts: async () => ['0x1234512345123451234512345123451234512345'],
+      sendTransaction: async () => {
+        return {
+          transactionHash:
+            '0x1234512345123451234512345123451234512345123451234512345123451234',
+        }
+      },
+    },
+  }
+
+  const apmStub = sinon.stub()
+  apmStub.returns({
+    getRepository: async () => {
+      return {
+        options: { address: '0x1234512345123451234512345123451234512345' },
+      }
+    },
+  })
+
+  const aclStub = sinon.stub()
+  aclStub.returns({
+    grant: () => {
+      return {}
+    },
+  })
+
+  const grantNewVersionsPermission = proxyquire
+    .noCallThru()
+    .load('../../../src/lib/apm/grantNewVersionsPermission', {
+      '@aragon/apm': apmStub,
+      './util/acl': aclStub,
+    })
+
+  t.context = {
```

Comment:
We are trying to move away from the context pattern introduced by `yargs` that makes code more difficult to type are reason about. It would be best to declare the variables with `let` in the module scope, assign them here and then use them on each test.

Reply:
Agree, we can do that with all tests on the next iteration.

Reply:
Sounds good